.github/workflows/mirror-images.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           path: vets-website
 
       - name: 'Download artifact'
+        if: ${{ github.event_name == 'workflow_run' }}
```

Comment:
This and follows: only do the artifact download/unzip/read if we actually expect it to be there.